composer.json

```diff
@@ -14,11 +14,11 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0"
+        "php": ">=5.3"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.*",
-        "satooshi/php-coveralls": "0.*"
+        "phpunit/phpunit": "^4.0|^5.0",
```

Comment:
A bit torn between the versions here. Imho it should be either version 4 or 5, not both. You could add a php5.6 requirement for dev here and then jump to phpunit 5, which I think is completely legit as it's dev only.